Modules/clinic/_tkinter.c.h

```diff
@@ -645,6 +652,65 @@ _tkinter_tkapp_willdispatch(TkappObject *self, PyObject *Py_UNUSED(ignored))
     return _tkinter_tkapp_willdispatch_impl(self);
 }
 
+PyDoc_STRVAR(_tkinter_tkapp_dispatching__doc__,
+"dispatching($self, /)\n"
+"--\n"
+"\n"
+"Returns the internal dispatching state\n"
+"\n"
+"Returns True if the main loop is running.\n"
+"Returns False if the main loop is not running.\n"
+"\n"
+"NOTE: Using update or dooneevent will dispatch events without the main\n"
+"      loop. Dispatching will return False in these cases.");
+
+#define _TKINTER_TKAPP_DISPATCHING_METHODDEF    \
+    {"dispatching", (PyCFunction)_tkinter_tkapp_dispatching, METH_NOARGS, _tkinter_tkapp_dispatching__doc__},
+
+static PyObject *
+_tkinter_tkapp_dispatching_impl(TkappObject *self);
+
+static PyObject *
+_tkinter_tkapp_dispatching(TkappObject *self, PyObject *Py_UNUSED(ignored))
+{
+    return _tkinter_tkapp_dispatching_impl(self);
+}
+
+PyDoc_STRVAR(_tkinter_tkapp_setmainloopwaitattempts__doc__,
+"setmainloopwaitattempts($self, new_val, /)\n"
+"--\n"
+"\n"
+"Set number of 100 millisecond mainloop wait attempts.\n"
+"\n"
+"These are used for threads that call(), var_invoke(), and createcommand().\n"
+"\n"
+"Current default is 10 for a 1 second wait, but future behavior\n"
+"will be equivalent to an unlimited number.\n"
+"\n"
+"Setting this will trigger a DeprecationWarning.");
+
+#define _TKINTER_TKAPP_SETMAINLOOPWAITATTEMPTS_METHODDEF    \
+    {"setmainloopwaitattempts", (PyCFunction)_tkinter_tkapp_setmainloopwaitattempts, METH_O, _tkinter_tkapp_setmainloopwaitattempts__doc__},
+
+static PyObject *
+_tkinter_tkapp_setmainloopwaitattempts_impl(TkappObject *self, int new_val);
+
+static PyObject *
+_tkinter_tkapp_setmainloopwaitattempts(TkappObject *self, PyObject *arg)
+{
+    PyObject *return_value = NULL;
+    int new_val;
+
+    new_val = _PyLong_AsInt(arg);
```

Comment:
```suggestion
    int new_val = _PyLong_AsInt(arg);
```
(removing previous `int new_val`)

Reply:
That bit was created by clinic, is it ok to mess with it?

Reply:
If it was created by clinic, that's fine (just wanted to make it a little neater if possible)